go.mod

```diff
@@ -6,10 +6,10 @@ toolchain go1.22.5
 
 require (
 	github.com/AlekSi/pointer v1.2.0
+	github.com/FerretDB/wire v0.0.4-0.20240729011506-6b6160b9bdbe
```

Comment:
It uses https://github.com/FerretDB/wire/pull/8 PR. Let's update it when this is merged.